### PR TITLE
Fix/lema/dt card header title is hidden on mobile phone

### PR DESCRIPTION
**Pull request by heartdisease**
### <strong>Pull Request</strong>

<hr>
Hi, thank you for contributing to Barista with this pull request (PR).

To ensure a fast process and merging of your PR please make sure it fulfills the
coding standards and contribution guidelines.

- A feature proposal has been provided, discussed and approved first.
- There is a meaningful description of the issue in GitHub (Screenshots are
  often helpful).
- If the PR introduces breaking-changes or deprecations it matches the following
  guidelines.
  - The commit message follows our commit guidelines.
  - Tests for the changes have been added (for bug fixes / features).
  - Docs have been added / updated (for bug fixes / features).

Please choose the type appropriate for the changes below: <br>

#### Checklist

- [ ] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by heartdisease** [comment]
Cherry-picked into:
10.7.x (5c349e4f0)
a11y-fixes-barista-11 (7ac4a36ac)

File: apps/dev/src/card/card-demo.component.html [hunk 1]
```diff
@@ -4,7 +4,6 @@
     <dt-card-subtitle>Some subtitle</dt-card-subtitle>
     <dt-card-title-actions>
       <button dt-button variant="secondary">Some Action</button>
-      <button dt-button variant="secondary">Action</button>
```
Comment (Sherif-Elhefnawy) [inline apps/dev/src/card/card-demo.component.html:7]: Please add the following back:
` button dt-button variant="secondary">Action</button>`

File: apps/dev/src/card/card-demo.component.html [hunk 2]
```diff
@@ -21,3 +20,23 @@
     </dt-card-footer-actions>
   </dt-card>
 </div>
+
+<div class="demo-card">
```
Comment (Sherif-Elhefnawy) [inline apps/dev/src/card/card-demo.component.html:24]: Thank you for adding the extra story

File: apps/dev/src/card/card-demo.component.html [hunk 3]
```diff
@@ -21,3 +20,23 @@
...
+  <dt-card>
+    <dt-card-title
+      >Top 3 JavaScript errors and lorem ipsum bla bla bla bla bla foo bar and
+      so on etc.
+    </dt-card-title>
+    <dt-card-subtitle
+      >Some subtitle lorem ipsum bla bla bla bla bla foo bar and so on etc.
+    </dt-card-subtitle>
+    <dt-card-title-actions>
+      <button dt-button variant="secondary">Some Action</button>
```
Comment (Sherif-Elhefnawy) [inline apps/dev/src/card/card-demo.component.html:34]: Could you please add the following:
`<button dt-button variant="primary">Action</button>`

**Review by Sherif-Elhefnawy — APPROVED** [review]

**Pull request merged**